pyproject.toml

```diff
@@ -42,7 +42,7 @@ dependencies = [
     # for logging, tracebacks, printing, progressbars
     "rich <= 13.0.1",
     # for CLI
-    "typer >= 0.6.0, < 1.0.0",
+    "typer >= 0.6.0, < 0.8.0",
```

Comment:
Is there any reason for this pin? If so, could you add an in-line comment to get to know in case we decide to upgrade at some point? Thanks 😄